posts/interaction-media-features.md

```diff
@@ -0,0 +1,11 @@
+feature: Interaction media features (hover, pointer, any-hover, any-pointer)
```

Comment:
I think we can just go with `Interaction media features` in here. No real reason to sum them up.